src/Adapter/Form/ChoiceProvider/CustomerThreadStatusesChoiceProvider.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Adapter\Form\ChoiceProvider;
+
+use PrestaShop\PrestaShop\Core\Domain\CustomerService\ValueObject\CustomerThreadStatus;
+use PrestaShop\PrestaShop\Core\Form\ConfigurableFormChoiceProviderInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class CustomerThreadStatusesChoiceProvider implements ConfigurableFormChoiceProviderInterface
```

Comment:
If you don't need to pass any options or parameters then the `FormChoiceProviderInterface` is enough

Reply:
```suggestion
class CustomerThreadStatusesChoiceProvider implements FormChoiceProviderInterface
```

Reply:
well when i change it to formChoice i get this error:
![image](https://user-images.githubusercontent.com/51944825/198289842-ff5bcd86-c5ae-4730-b548-6518b409ebc7.png)

Reply:
Yeah because you probably didn't update the grid factory that relies on it